examples/attacks.py

```diff
@@ -265,3 +265,29 @@ def identity(
         tensor: torch.Tensor, mask: tp.Optional[torch.Tensor] = None
     ) -> tp.Union[tp.Tuple[torch.Tensor, torch.Tensor], torch.Tensor]:
         return audio_effect_return(tensor=tensor, mask=mask)
+
+    @staticmethod
+    def shush(
+        tensor: torch.Tensor,
+        fraction: float = 0.001,
+        mask: tp.Optional[torch.Tensor] = None
+    ) -> tp.Union[tp.Tuple[torch.Tensor, torch.Tensor], torch.Tensor]:
+        """
+        Sets a specified chronological fraction of indices of the input tensor (audio signal) to 0.
+
+        Parameters:
+        - tensor (torch.Tensor): Input audio tensor. Assumes tensor shape is (batch_size, channels, time).
+        - fraction (float): Fraction of indices to be set to 0 (from the start of the tensor) (default: 0.001, i.e, 0.1%)
+
+        Returns:
+        - torch.Tensor: Transformed audio tensor.
+        """
+        batch_size, channels, time = tensor.shape
+        shush_tensor = tensor.detach().clone()
+        
+        # Set the first `fraction*time` indices of the waveform to 0
+        for b in range(batch_size):
+            for channel in range(channels):
+                shush_tensor[b][channel][0:int(fraction*time)] = 0.0
```

Comment:
```suggestion
        time = tensor.size(-1)
        shush_tensor = tensor.detach().clone()
        
        # Set the first `fraction*time` indices of the waveform to 0
        shush_tensor[:, :, :int(fraction*time)] = 0.0
```

Reply:
This has been added in https://github.com/facebookresearch/audioseal/pull/25/commits/f39691178b36963665a50c2a7d1f98acf9dddc7e